server/data-import/data.js

```diff
@@ -25,23 +22,36 @@ loadData()
 
 // note that exports are 'mongoexport --db hah-dev --collection pregnancycenters --jsonArray --out cessilye_nypc_geocoded.json'
 
-PregnancyCenterModel.collection.dropAsync()
-	.catch( (err) => log.error(err))
-	.then( () => {
-		fs.readFile('test/fixtures/cessilye_nypc.json', 'utf8', (err, data) => {
-			const docs = EJSON.parse(data)
-			log.info(typeof docs)
-			for (const doc of docs) {
+async function reimport() {
 
-				Joi.validateAsync(doc, pregnancyCenterSchemaJoi, {abortEarly: false})
-					.catch( (err) => log.error(err))
-					.then( (validatedData) => {
-						PregnancyCenterModel.createAsync(validatedData)
-							.then( (pc) => log.info(pc))
-					})
+	let docs
 
-			}
+	try {
+		await PregnancyCenterModel.collection.drop()
```

Comment:
If this does fail, do we want the script to keep running? I'm thinking we would probably want it to fail so that we don't have duplicate data and then we could figure out why it failed before running again? I believe that catching the error and then logging it would allow the script to keep running, but I could be wrong on that.